input/WIxSC_0.yml

```diff
@@ -0,0 +1,23 @@
+mapper_class: MapperWIxSC
+mask_name: mask_2MPZWISC
+path_rerun: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/xcell_runs'
+coords: G
+z_edges: [0.10, 0.15]
+bin_name: 'z0p10_0p15'
```

Comment:
maybe enumerate the possible bin names in the mapper for user clarity?

Reply:
The trouble is that these N(z)s will be saved to rerun files. So if we change the definition of the bin edges, we run the risk of reading the rerun files from the previous run with different bins, rather than recalculating them. This ensures that all distinct bins will be saved in different files.